### ISSUE TITLE
HtmlMin should not decode reserved characters entities (&lt;, &gt;)

### ISSUE THREAD

**Issue by jsamr** [voku/HtmlMin#22]
#### How can I reproduce it?

The issue happens with `&lt;` and `&gt;`.

``` php
<?php

use voku\helper\HtmlMin;

$minifier = new HtmlMin();
echo $minifier->minify('<span>&lt;</span>');
// prints:  <span><</span>
// expects: <span>&lt;</span>
```
However, a lonely entity is **not** decoded.

``` php
<?php

use voku\helper\HtmlMin;

$minifier = new HtmlMin();
echo $minifier->minify('&lt;');
// prints:  &lt;
// expects: &lt;
```

#### Does it take minutes, hours or days to fix?

I would say minutes.

#### Any additional information?

Voku version 3.0.4

**Comment by voku** [comment]
Thanks for the bug-report, fixed in version 3.0.5.

**Comment by jsamr** [comment]
@voku Thanks a lot, that was fast!
